src/main/kotlin/uk/gov/dluhc/printapi/mapper/CertificateMapper.kt

```diff
@@ -27,7 +27,7 @@ abstract class CertificateMapper {
     ): Certificate
 
     @AfterMapping
-    protected fun addPrintRequestToCertificate(
+    fun addPrintRequestToCertificate(
```

Comment:
AfterMapping should be protected, if you wish to use it, then you could add another public fun that eventually after maps and uses same protected method.

Reply:
Minor (x2) - I'm not sure why the fun has been made `public`, but I also don't fully follow what @vishalgupta-valtech is saying about different patterns for this use case?

Reply:
Was not aware MapStruct methods annotated as `@AfterMapping` should not be public - was made public so could use the method in `uk.gov.dluhc.printapi.service.PrintService`.

Will refactor to have `PrintService` use the `PrintRequestMapper` and then call the `certificate.addPrintRequest` method.